com.github.tmewett.BrogueCE.appdata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
The file misses OARS and developer name.

Reply:
Ah yes i will generate one of those also